qldbshell/qldb_shell.py

```diff
@@ -142,7 +149,7 @@ def do_EOF(self, line):
 
     @zero_noun_command
     def quit_shell(self, line):
-        logging.info("Exiting qldb shell.")
+        print("Exiting qldbshell.")
```

Comment:
"Exiting QLDB Shell.". Be consistent on the tool name.

Reply:
Updated.